src/components/views/rooms/wysiwyg_composer/components/WysiwygComposer.tsx

```diff
@@ -72,10 +74,10 @@ export const WysiwygComposer = memo(function WysiwygComposer({
     useSetCursorPosition(!isReady, ref);
 
     useEffect(() => {
-        if (!disabled && content !== null) {
-            onChange?.(content);
+        if (!disabled && isNotNull(messageContent)) {
+            onChange(messageContent);
         }
-    }, [onChange, content, disabled]);
+    }, [onChange, messageContent, disabled]);
```

Comment:
This is the change that ensures our message content, when sent, is correctly formatted according to the matrix spec as opposed to being exactly the markup used in the html inside the composer.